File: WdtcCore/src/main/java/org/wdt/wdtc/core/download/infterface/ModDownloadInfoInterface.java
package org.wdt.wdtc.core.download.infterface;

import org.wdt.wdtc.core.utils.ModUtils;

public interface ModDownloadInfoInterface {

    String getModVersion();

    InstallTaskInterface getModInstallTask();

    ModUtils.KindOfMod getModKind();

}

File: WdtcCore/src/main/kotlin/org/wdt/wdtc/core/download/infterface/ModDownloadInfoInterface.kt
package org.wdt.wdtc.core.download.infterface

import org.wdt.wdtc.core.utils.KindOfMod

interface ModDownloadInfoInterface {
  val modVersion: String
  val modInstallTask: InstallTaskInterface
  val modKind: KindOfMod
  override fun toString(): String
}
